src/Service/ExcelImport.php

```diff
@@ -266,4 +268,18 @@ private function getCellValueOrNull(Worksheet $sheet, int $col, int $row)
 
         return $cell->getValue();
     }
+
+    private function checkIfItemExists($identifier, &$items)
+    {
+        $item = $this->getEntityManager()
+            ->getRepository(LsItem::class)
+            ->findOneBy(['identifier' => $identifier]);
+
+        if ($item !== null) {
+            $items[$item->getIdentifier()] = $item;
```

Comment:
the `getItemByIdentifer` function should be in the same controller or in the LsItem Repository?

Reply:
It would be in the controller, or you can just use the repository call that it contains directly in the location that you are calling it.

`$this->getEntityManager()->getRepository(LsItem::class)->findOneBy(['identifier' => $identifier]);`
is the same as
`$this->getEntityManager()->getRepository(LsItem::class)->findOneByIdentifier($identifier);`

so you could add a `$itemRepo = $this->getEntityManager()->getRepository(...)` earlier in the calling method and then just use `if ($item = $itemRepo->findOneByIdentifier($identifier))...` if you wanted.